### ISSUE TITLE
Create an exception if a request is getting blocked

### ISSUE THREAD

**Issue by desecho** [LevPasha/Instagram-API-python#349]
If a request is getting blocked, we get this kind of response - {'message': 'Sorry, there was a problem with your request.', 'status': 'fail', 'error_type': 'sentry_block'}
We should raise an exception in this case I think. I can create a PR if you'd like.

**Comment by Siralland01** [comment]
It seems like Instagram has attempted to block the bots. I tried changing the user agent but I've had no luck with that. I've also changed my ip and that doesn't work either. Still looking

**Comment by desecho** [comment]
I created an issue because I think it will be nice to have an exception raised if this happens, Currently it just fails silently which causes all sorts of issues. 

Concerning the error itself, you can read about on the faq page of the original php version (on github)

In my experience, after some time they unblock you.

**Comment by Siralland01** [comment]
How are they blocking me though? My account isn't banned and I changed my IP. What else can they identify the bot with?

**Comment by desecho** [comment]
I don’t know how they do it, but for me - it was working from one set of ips and not working for the other set of ips at the same time. So it is unclear how they do it.

**Comment by akuma34** [comment]
i am also figuring it out, for some IP it is working for me, for new IPs it is not working for me.

**Comment by desecho** [comment]
See my commit to attempt to avoid getting blocked https://github.com/desecho/ighelper/commit/a44bac6dc06a24865128aea5bedbcf989b2b8937

I am not sure if it is going to work though 😢

**Comment by desecho** [comment]
So currently for me it seems to work intermittently with the same IP so.. not sure what is going on there.

**Comment by galactic16** [comment]
I just tried different things : 
- A never-used account on one of my IP address that was blocked : sentry_block
- A never-used IP address with one of my old account : sentry_block
- This one is so weird : I ran a simple login script with a totally new account on a totally new IP address and I got blocked right away. I'm starting to think that it could be the python API that is blocked?..
I'm starting to feel a bit desperate...

**Comment by joselitinho1** [comment]
@galactic16 are you using residential IPs or from cloud?

**Comment by galactic16** [comment]
@joselitinho1 I was using it from a VPS so from cloud yeah, I think that's why. Re-tested it today and still blocked, I think it's gone for good...
